pkg/controller/migmigration/task.go

```diff
@@ -427,6 +432,7 @@ func (t *Task) Run() error {
 		status, err := t.runHooks(migapi.PostBackupHookPhase)
 		if err != nil {
 			log.Trace(err)
+			t.fail(PostBackupHooksFailed, []string{err.Error()})
```

Comment:
@jortel is this the right way to move a migration to failed state? I don't know the convention for this, haven't had to send a migration to failure in code I've written.

Reply:
When I asked this is what I was pointed at. It seems to work well like this. We still get the trace and the migration fails with the condition/error that specifically caused the issue.

Reply:
Cool, that sounds reasonable.